plexapi/playlist.py

```diff
@@ -54,7 +58,10 @@ def _loadData(self, data):
         self.icon = data.attrib.get('icon')
         self.key = data.attrib.get('key', '').replace('/items', '')  # FIX_BUG_50
         self.leafCount = utils.cast(int, data.attrib.get('leafCount'))
+        self.librarySectionKey = data.attrib.get('librarySectionKey')
```

Comment:
Added `librarySectionID` to doc string but missing attribute?
```suggestion
        self.librarySectionID = utils.cast(int, data.attrib.get('librarySectionID'))
        self.librarySectionKey = data.attrib.get('librarySectionKey')
```

Reply:
This is already set elsewhere, but it _is_ included in the playlist XML payload. Better to set it explicitly like you suggest. 👍